workspaces/src/network/server.rs

```diff
@@ -25,6 +25,10 @@ impl SandboxServer {
     }
 
     pub fn start(&mut self) -> anyhow::Result<()> {
+        if self.process.is_some() {
+            anyhow::bail!("Sandbox server already started");
+        }
```

Comment:
huh, what case were you hitting where `start` was being called multiple times? Still good to have this in here just in case though

Reply:
No, but reading the logic it seemed like it could be a hidden bug if logic changed external to this method